items.json

```diff
@@ -3512,42 +3512,6 @@
       }
     ]
   },
-  {
-    "id": "crpg_battania_axe_3_t5",
-    "name": "Round Bitted Fine Steel Hatchet",
-    "culture": "Battania",
-    "type": "OneHandedWeapon",
-    "price": 2875,
-    "weight": 1.03,
-    "tier": 5.77098036,
-    "requirement": 0,
-    "flags": [
-      "CanBePickedUpFromCorpse"
-    ],
-    "weapons": [
-      {
-        "class": "OneHandedAxe",
-        "itemUsage": "onehanded_shield_axe",
-        "accuracy": 0,
-        "missileSpeed": 0,
-        "stackAmount": 0,
-        "length": 80,
-        "balance": 0.7,
-        "handling": 88,
-        "bodyArmor": 0,
-        "flags": [
-          "MeleeWeapon",
-          "BonusAgainstShield"
-        ],
-        "thrustDamage": 14,
-        "thrustDamageType": "Pierce",
-        "thrustSpeed": 95,
-        "swingDamage": 29,
-        "swingDamageType": "Cut",
-        "swingSpeed": 90
-      }
-    ]
-  },

```

Comment:
You forgot to add this to available/usablepiece

Reply:
got it fixed I believe